docs/contribute_to_doc_guide.rst

```diff
@@ -0,0 +1,26 @@
+===================================
+Contributing to the documentation
+===================================
+This is to provide some guidance on contributing to setuptools'
+documentation for those unfamiliar with sphinx. setuptools' documentation is built using 
+`Sphinx <https://pypi.org/project/Sphinx/>`_ and written in `reStructuredText <https://docutils.sourceforge.io/rst.html>`_.
+
+First navigate to the ``docs/`` folder. Some dependencies are required to build the 
+documentation::
+
+  pip install -r requirements.txt
+
+Then, invoke the ``make`` program to build the documentation in the desired
+format. For example::
+
+  make html
+
+.. note::
+  Learn more about the `make program <https://en.wikipedia.org/wiki/Make_(software)>`_
+  and `pip install <https://pip.pypa.io/en/stable/reference/pip_install/#id18>`_
```

Comment:
I'm surprised that `Makefile` still exists in that folder. Nobody uses that and it should probably be removed to avoid confusion as it's not being kept up-to-date anyway, tox is set up to invoke sphinx directly and RTD does the same...